server/service/osquery_utils/queries.go

```diff
@@ -1201,6 +1201,18 @@ func GetDetailQueries(fleetConfig config.FleetConfig, features *fleet.Features)
 		}
 	}
 
+	if features != nil {
+		for name, override := range features.DetailQueryOverrides {
+			if override == nil {
+				delete(generatedMap, name)
+			} else {
+				query := generatedMap[name]
```

Comment:
`query, ok := generatedMap[name]`

What should we do `if !ok`? (e.g. a typo in the query name in the yaml)